src/class-elements/syntax/valid/cls-decl-elements-valid-syntax-inside-function.template

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2019 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: prod-ClassElement
+name: class declaration module inside function
+path: language/statements/class/elements/syntax/valid/inside-function-
+features: [class]
+---*/
+
+function getC() {
+  class C /*{ heritage }*/{
+    /*{ elements }*/
+  }
+  return C;
+}
+var C = getC();
```

Comment:
I believe this is really excessive. A single test to verify the production of a class inside a function would me more than enough.

Also, the `var C = getC();` is going beyond a test for valid syntax.

I recommend removing this.